doc/040_backup.rst

```diff
@@ -45,6 +45,17 @@ size of the files and directories in ``~/work`` on the local file system. It
 also tells us that only 1.200 GiB was added to the repository. This means that
 some of the data was duplicate and restic was able to efficiently reduce it.
 
+We just attached the absolute path ``~/work`` to the backup, so the path
+within the repository is ``/home/user/work``, depending on your user name.
```

Comment:
Paths only exist within snapshots, not the repository.

Reply:
This is another argument why snapshots should list the attached path and the "path within the snapshot". And maybe inner path is a shorter name